src/robot_arm.hpp

```diff
@@ -1,19 +1,30 @@
-#ifndef ROBOT_ARM_HPP
-#define ROBOT_ARM_HPP
+#ifndef ROBOTARM_HPP
+#define ROBOTARM_HPP
 
-#include "swiftpro/"
+#include "functions.hpp"
 #include "wrap-hwlib.hpp"
 
-class robot_arm {
+class RobotArm {
   private:
+    Functions f;
     int current_coordinates[3];
     int goto_coordinates[3];
     hwlib::target::pin_out output_pin;
 
   public:
-    robot_arm(hwlib::target::pin_out output_pin);
-    void move(int coordinates[3]);
-    void copy_data(int *array1, int *array2);
+    /**
+     * @brief Construct a new Robot Arm object
+     *
```

Comment:
er zijn geen descriptions waar in zou komen te staan dat die voor nu alleen nog maar de coordinaten print